cookbooks/aws-parallelcluster-slurm/recipes/update/update_head_node.rb

```diff
@@ -93,15 +93,25 @@ def get_all_queues(config)
   end
 
   def get_queues_with_changes(config)
```

Comment:
same here, we can unit test this function mocking external logic.

Reply:
Good point, thanks. I assumed it was not. I will include it in the next iteration.